app/Http/Controllers/PIM/EmployeeListController.php

```diff
@@ -95,10 +95,10 @@ public function store(PIMRequest $request)
     {
         try
         {
-            $temp = $this->employee->create($request->all());
+            $new_user = $this->employee->create($request->all());
```

Comment:
it's not a new_user you are creating. It's a new employee. change your variable.